Makefile

```diff
@@ -23,10 +23,57 @@ $(MAKEFILE):
 
 -include $(MAKEFILE)
 
+# Conf vars
+GITHUB_API_KEY ?=
+LOGS_PATH ?= /var/log/docsrv
+LANDING_PATH ?= $(SHARED_PATH)/landing
+INIT_PATH ?= $(shell pwd)/../ci/docs/init.d
```

Comment:
Using `shell pwd` here instead of a variable such as `SHARED_PATH` seems suspicious.